content/source/docs/enterprise/install/active-active.html.md

```diff
@@ -357,7 +355,7 @@ For example:
    
    "user_token":{
       "value":"6c403112981d0ae2ce6b4b2c7a1b1997"
-   }
+   },
```

Comment:
The trailing comma after the "registry_session_secret_key" should be removed, per the JSON specification. Can't add this comment to that line, though 🤷🏻

Reply:
And you do syntax checking! corrected.